<launch>
  <arg name="myVar1_" default="0" doc="explain this variable"/>
  <arg name="myVar2_" default="5.0" doc="explain this variable"/>
  <arg name="timerFreq_" default="20.0" doc="explain this variable"/>
  <arg name="odom_topic" default="/odom" doc="explain this variable"/>
  <arg name="imu_topic" default="/imu/data" doc="explain this variable"/>
  <arg name="vehicle_cmd_topic" default="/vehicle_cmd" doc="explain this variable"/>
  <arg name="vehicle_status_topic" default="/vehicle_status" doc="explain this variable"/>

  <node pkg="template_ros_node" type="simple_node" name="simple_node" output="screen">
    <param name="myVar1_" value="$(arg myVar1_)" type="int"/>
    <param name="myVar2_" value="$(arg myVar2_)" type ="double"/>
    <param name="timerFreq_" value="$(arg timerFreq_)" type ="double"/>
    <remap from="/odom" to="$(arg odom_topic)"/>
    <remap from="/imu/data" to="$(arg imu_topic)"/>
    <remap from="/vehicle_cmd" to="$(arg vehicle_cmd_topic)"/>
    <remap from="/vehicle_status" to="$(arg vehicle_status_topic)"/>
  </node>
</launch>